tasks.rb

```diff
@@ -401,6 +404,8 @@ def create_hpxmls
     'base-hvac-undersized.xml' => 'base.xml',
     'base-hvac-undersized-allow-increased-fixed-capacities.xml' => 'base-hvac-undersized.xml',
     'base-hvac-wall-furnace-elec-only.xml' => 'base.xml',
+    'base-hvac-ptac-electric-resistance.xml' => 'base.xml',
```

Comment:
I suggest splitting this up into two files:
- base-hvac-ptac.xml
- base-hvac-ptac-cooling-only.xml

Since the AHRI directory showed that the vast majority of PTACs include electric resistance heating, I'm suggesting that that be assumed in the base-hvac-ptac.xml file.

Reply:
I am hoping that the PTAC with cooling only will not need a `HeatingSystem` defined in the HPXML, is that right? (In contrast to forcing the user to have to create such a system but with capacity and fraction load served set to zero.)